src/main/java/lindar/fonix/api/FonixSmsResource.java

```diff
@@ -180,7 +180,7 @@ public DeliveryReport parseDeliveryReport(Map<String, String> mapParameters) {
             if (mapParameters.containsKey(DR_STATUS_TIME)) {
                 deliveryReport.setStatusTime(parseStatusTime.parse(mapParameters.get(DR_STATUS_TIME)));
             }
-        } catch (ParseException e) {
+        } catch (ParseException | NumberFormatException e) {
```

Comment:
There are more catch statements in this file, maybe it makes sense to update all of them.